shared/fs/nav-header/mobile-header.js

```diff
@@ -58,13 +58,19 @@ class MobileHeader extends React.PureComponent<Props, State> {
   }
 }
 
+const statusBarHeight = isIPhoneX ? 45 : 20
```

Comment:
I believe this is in styles already

Reply:
https://github.com/keybase/client/blob/3e6b7e6ebd9598f34d53c6d7c3034009b5368d0c/shared/styles/index.native.js#L51

doesn't seem to consider iPhoneX yet, so I added it to the new commit.

Reply:
(talked on chat; rebased over nojima's branch)